numba/targets/arraymath.py

```diff
@@ -639,6 +645,17 @@ def check_blas_return(context, builder, res):
         pyapi.gil_ensure()
         pyapi.fatal_error("BLAS wrapper returned with an error")
 
+def check_lapack_return(context, builder, res):
```

Comment:
Perhaps combine with `check_blas_return` and add the error string as a `kwarg`? Both check for error on return from a function wrapping some "native" functions.